pkg/utils/impression.go

```diff
@@ -72,12 +72,19 @@ func CreateImpressionTrackingGoal(vwoInstance schema.VwoInstance, variationID in
 
 	impression.URL = constants.HTTPSProtocol + constants.EndPointsBaseURL + constants.EndPointsTrackGoal
 	impression.GoalID = goalID
-	if revenueGoal > 0 {
-		impression.R = revenueGoal
+
+	if goalType==constants.GoalTypeRevenue && revenueValue >= 0 {
```

Comment:
what if this is false?

Reply:
If GoalType is not REVENUE_TRACKING we dont need to address impression.R